app/scripts/components/sandbox/exploration-map/index.js

```diff
@@ -0,0 +1,114 @@
+import React from 'react';
+import styled from 'styled-components';
+import Constrainer from '$styles/constrainer';
+import { PageMainContent } from '$styles/page';
+import { ExplorationMap } from '$components/exploration/components/map';
+
+const DemoExplorationMap = styled(ExplorationMap)`
+  height: 40rem;
+`;
+
+const Wrapper = styled.div`
+  position: relative;
+  grid-column: 1 / -1;
+  height: 42rem;
+`;
+
+const mockDatasets = [
```

Comment:
I'm not sure if this is a nitpick or not. But any reason that you are passing the incomplete dataset here? (This data does not seem to be enough to render the actual layer.)

Reply:
Thanks for catching this! The sandbox should load the raster layers correctly now.